homework3/tests/test_task04.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+
+from homework3.task04.task04 import is_armstrong
+
+
+@pytest.mark.parametrize("number", [2, 7, 153, 371])
+def test_is_armstrong_true(number):
+    assert is_armstrong(number) is True
+
+
+@pytest.mark.parametrize("number", [10, 66, 999])
```

Comment:
You should add a negative number to this test since you add the check in the function.